packages/extras/src/lib/components/Float/Float.svelte

```diff
@@ -1,89 +1,83 @@
 <script lang="ts">
-  import { Group as ThreeGroup, MathUtils } from 'three'
-  import { useFrame, Group } from '@threlte/core'
-  import type { FloatProperties } from '../../types/components'
+  import { MathUtils } from 'three'
+  import { useFrame, T } from '@threlte/core'
+  import type { FloatProps } from './Float.svelte'
 
-  // Group Properties
-  export let position: FloatProperties['position'] = undefined
-  export let scale: FloatProperties['scale'] = undefined
-  export let rotation: FloatProperties['rotation'] = undefined
-  export let lookAt: FloatProperties['lookAt'] = undefined
-  export let viewportAware: FloatProperties['viewportAware'] = false
-  export let inViewport: FloatProperties['inViewport'] = false
-  export let castShadow: FloatProperties['castShadow'] = undefined
-  export let receiveShadow: FloatProperties['receiveShadow'] = undefined
-  export let frustumCulled: FloatProperties['frustumCulled'] = undefined
-  export let renderOrder: FloatProperties['renderOrder'] = undefined
-  export let visible: FloatProperties['visible'] = undefined
-  export let userData: FloatProperties['userData'] = undefined
-  export let dispose: FloatProperties['dispose'] = undefined
+  type Props = Required<FloatProps>
```

Comment:
Add the types for events and slots:

```ts
type $$Events = FloatEvents
type $$Slots = FloatSlots
```

This will ensure that the Svelte LSP can pick up on mistakes in the implementation. Currently this is especially useful on slot types.